IronFoundry.Container.Shared/Utilities/FileSystemEffectiveAccessComputer.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Security.AccessControl;
+using System.Security.Principal;
+
+namespace IronFoundry.Container.Utilities
+{
+    public class FileSystemEffectiveAccessComputer
+    {
+        public FileSystemRights ComputeAccess(string path, IdentityReference identity)
+        {
+            var reader = new FileSystemSecurityDescriptorReader(path);
+            var descriptor = reader.GetSecurityDescriptor();
+            var effectiveAccess = new EffectiveAccessComputer();
+            var access = effectiveAccess.ComputeAccess(descriptor, identity);
+            var rights = (FileSystemRights) access;
```

Comment:
If we are going to cast the `ACCESS_MASK` to a `FileSystemRights`, why not just return `FileSystemRights` directly?